pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -299,14 +291,14 @@ func (p *PriorityQueue) AddUnschedulableIfNotPresent(pInfo *framework.QueuedPodI
 	defer p.lock.Unlock()
 	pod := pInfo.Pod
 	if p.unschedulableQ.get(pod) != nil {
-		return fmt.Errorf("pod: %v is already present in unschedulable queue", nsNameForPod(pod))
+		return fmt.Errorf("pod %v is already present in unschedulable queue", klog.KObj(pod))
```

Comment:
```suggestion
		return fmt.Errorf("Pod is already present in unschedulable queue", "pod", klog.KObj(pod))
```

Reply:
Done.

Reply:
Errorf formats according to a format specifier and returns the string as a value that satisfies error.
https://golang.org/pkg/fmt/#Errorf

I think it's better to keep the original modification.